cabal.project

```diff
@@ -1,4 +1,4 @@
-index-state: 2021-02-15T00:00:00Z
+index-state: 2021-11-23T00:00:00Z
```

Comment:
The `master` branch of `cardano-ledger-specs` is currently at 12-03:06:23Z

https://github.com/input-output-hk/cardano-ledger/blob/80d2624174b3fc598c95ab9bf0a21c9bc7e045d3/cabal.project#L1

My intuition is that we'll benefit from having the same index-state as our primary upstream library. Would that work here?

(I made the same comment on PR #3530.)